about.html

```diff
@@ -0,0 +1,159 @@
+                  fill="white"
+                />
+              </svg>
+            </span>
+             
+        <a href="index.html" class="nav-link">Home</a>
+        <a href="#" class="nav-link">program</a>
+        <a href="#" class="nav-link">join</a>
+        <a href="#" class="nav-link">sponser</a>
+        <a href="#" class="nav-link">news</a>
+
+           </div>
+            
+            
+        </nav>
+</header>
+   
+   
+   <div class="content">
+    <div class="head">
+      <p class="la red Hello">"Hello, tech womens"</p>
+      <h1 class="co red headp"> Web programming courses for creative and busy women 2023 </h1>
+    </div>
+    <div class="para1 co">
+      <P> 
+        The Global summit is an international nonprofit dedicated to inspiring women to excel in technology careers.#women who code is here to help bridge the gender gap in the tech programming industry by providing knowledge that is becoming necessary in today’s increasingly technological job market.
+        The Summit brings together the community of exeperts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals.
+      </P>
+    </div>
+    <div class="para2 la">
+           <p class=" para2-first la"> Please contact us per Email for any further questions about web development 2023  </p>
+            <p class="second la"><strong><a href="#">beteland23@gmail.com</a></strong></p>
+    </div>
+   
+   </div>
+   </section>
+   <section class="container-back">
+   <section class="container la">
+    <p class="strong"><strong> Creative Common Global summit 2015 logo</strong>  </p>
+    <hr>
+    <p class="light"> The logo of 2015 world of women coders is desided form the logo compition from 23 june to 30.
+    </p>
+    <div class="image">
+        <img src="img/logo.PNG" alt="logo">
+    </div>
+   </section>
+  </section>
+   <section class="past-summit">
+    <p class="str la"><strong> See the past cc Global Summit</strong>
+      </p>
+      <hr>
+    <p class="lig la none"> take look at the last two CC gloabla summit which took place </p>
+    <p class="la cut none"> in Bueno sares and Addis Ababa</p>
+    <p class="one la">take look at the last two CC gloabla summit which took place in Bueno sares and Addis Ababa</p>
+    <section class="history">
+        <div class="first-histo">
+            <strong class="year la">2022</strong>
+            <p class="la cc">CC Global summit 2022 in Addis Ababa</p>
+        </div>
+        <div class="sec-histo">
+            <strong class="year la">2022</strong>
+            <p class="la cc">CC Global summit 2022 in Bueno sares </p>
+        </div>
```

Comment:
- [ ] Kindly, make sure this section match the provided design. add a orange layer and center the text inside it as it's in the screen capture of the provided design.

**Your design**
![image](https://user-images.githubusercontent.com/74411135/235253643-4db4f676-e165-4dfd-8965-8832b44c8b43.png)

**Provided design**
![image](https://user-images.githubusercontent.com/74411135/235253696-c96175b7-ba93-4d97-948b-61b92f57ce0a.png)

This will make your project way more prettier which will improve the user experience 📱  😄 👍